src/main/java/org/broadinstitute/hellbender/cmdline/DeprecatedArtifactsRegistry.java

```diff
@@ -0,0 +1,57 @@
+package org.broadinstitute.hellbender.cmdline;
+
+import scala.Tuple2;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * When a tool is removed from GATK (after having been @Deprecated for a suitable period), an entry should
+ * be added to this list to issue a message when the user tries to run that tool.
+ *
+ * NOTE: Picard tools should be listed here as well, since by definition such tools will not be found in
+ * the Picard jar.
```

Comment:
Maybe a registry in Picard that can add on runtime values will be helpful. The same for dowsntream tools...

Reply:
If and when picard has such a thing we can consider integrating it here. for downstream toolkits, getUnknownCommand is overridable now, which should allow the flexibility to do whatever integration is necessary.